website/blog/2022-06-16-resources-migrating-your-react-native-library-to-the-new-architecture.md

```diff
@@ -0,0 +1,112 @@
+
+#### RNNewArchitectureLibraries
+
+Similarly, this repo provides a step-by-step guide about how to create a **TurboModule** and a **Fabric Component**. It has a focus on ensuring backward compatibility between the new architecture and legacy.
+
+The repository is organized in an similar way to the previous one:
+
+- A **main** branch has no code but a README.md which advertises other branches.
+- Other branches to show how to develop **TurboModules** and **Fabric Components**.
+
+We plan to keep this example updated onto new releases of React Native, especially releases that affect library development, as well as add more examples on how to use advanced features (for example: implementing commands, event emitters, custom state). If you notice errors, please file an issue in the example repository.
+
+### Support
+
+We’ve created a dedicated [working group](https://github.com/reactwg/react-native-new-architecture) to give the community space to ask questions and get updates on the New Architecture. If you are a library maintainer, this is a valuable resource to find answers to your questions, and for us to know about your requirements. To join, please follow [these instructions](https://github.com/reactwg/react-native-new-architecture#how-to-join-the-working-group). Everyone is welcome.
+
+The working group is organized in several sections:
+
+- [Announcements](https://github.com/reactwg/react-native-new-architecture/discussions/categories/announcements): A place to share milestones and significant updates on the RN New Architecture Rollout
+- [Deep Dive](https://github.com/reactwg/react-native-new-architecture/discussions/categories/deep-dive): A place to chat about deep dives and technical-specific topics
+- [Documentation](https://github.com/reactwg/react-native-new-architecture/discussions/categories/documentation): A place to chat about the New Architecture documentation and migration material
+- [Libraries](https://github.com/reactwg/react-native-new-architecture/discussions/categories/libraries): A place to chat about 3rd party libraries and their migration story to the New Architecture
+- [Q&A](https://github.com/reactwg/react-native-new-architecture/discussions/categories/q-a): A place to ask the community for help on the New Architecture topics
+- [Releases](https://github.com/reactwg/react-native-new-architecture/discussions/categories/releases): A place to chat about release specific bugs & build problems
+
+To use this group effectively:
+
+- **Make sure your library is listed inside the [Libraries] (https://github.com/reactwg/react-native-new-architecture/discussions/categories/libraries)section**. This will help us share a status update on the migration of your library and will help us understand which struggles library authors are facing to support you better.
```

Comment:
```suggestion
- **Make sure your library is listed inside the [Libraries](https://github.com/reactwg/react-native-new-architecture/discussions/categories/libraries) section**. This will help us share a status update on the migration of your library and will help us understand which struggles library authors are facing to support you better.
```
fix